Content.Server/Voting/VoteCommands.cs

```diff
@@ -129,6 +179,85 @@ public CompletionResult GetCompletion(IConsoleShell shell, string[] args)
             var n = args.Length - 1;
             return CompletionResult.FromHint(Loc.GetString("cmd-customvote-arg-option-n", ("n", n)));
         }
+
+        // Sends the payload's message. 
+        private async void WebhookMessage(WebhookPayload payload, string id)
+        {
+            if (id == string.Empty)
+            {
+                var request = await _httpClient.PostAsync($"{_webhookUrl}?wait=true",
+                    new StringContent(JsonSerializer.Serialize(payload), Encoding.UTF8, "application/json"));
+
+                var content = await request.Content.ReadAsStringAsync();
+                webhookId = (string) JsonNode.Parse(content)?["id"]!;
+            } 
```

Comment:
Remove it and make not null or empty check in the `Execute()` method

Reply:
This is so it'll send a new message if there is no ID, or edit the message if there is. It's intentionally going to be empty.